### PR TITLE
fix(paramount): fix presence

### PR DESCRIPTION
**Pull request by calicokatsu**
## Description 
- Escaped `if-else hell` (kinda)
- Fixed Live TV Presence
- Fixed Series Presence
- Series Presence now has the Episode Thumbnail as it's Large Image Key

## Acknowledgements
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`

## Screenshots
image
image

<details>
<summary> Series' have been reported to not be working. This push fixes that and the Series presence now also displays the episode thumbnail alongside the Title, Season and Episode. Along with fixing the Live TV Presence</summary>

</details>

### CODE REVIEW

**Comment by calicokatsu** [comment]
oops forgot to lint

File: websites/P/Paramount+/presence.ts [hunk 1]
```diff
@@ -22,132 +22,157 @@ presence.on("UpdateData", async () => {
...
+			presenceData.state = liveTitle;
+			presenceData.smallImageKey = Assets.Live;
+
+			break;
+
+		case pathIncludes(path, "/brands"):
+			if (path.includes("/cbs/")) {
+				presenceData.details = "Browsing Brand:";
+				presenceData.state = "CBS";
+				presenceData.largeImageKey = "https://i.imgur.com/BfxpVej.png";
```
Comment (theusaf) [inline websites/P/Paramount+/presence.ts:137]: You have many assets in this presence. Put them in an enum like so: https://github.com/PreMiD/Presences/blob/1f5a4117f92c6549a75e1c01c6305f9c4398b5b5/websites/A/Aodesu/presence.ts#L6-L14

**Review by theusaf — CHANGES_REQUESTED** [review]

**Comment by calicokatsu** [comment]
dang my checks are failing :(

File: websites/G/Gogoanime/metadata.json
Comment (theusaf) [inline websites/G/Gogoanime/metadata.json:1]: You have committed changes to presences unrelated to this PR. To resolve this issue, revert the changes made to these.

**Review by theusaf — CHANGES_REQUESTED** [review]

**Comment by calicokatsu** [comment]
I think this branch is just permanently tainted

**Comment by theusaf** [comment]
> I think this branch is just permanently tainted

You can fix this by doing something like:
```bash
git checkout main websites/S websites/V
```
(assuming your `main` branch is up to date with PreMiD's)
This will load the files from the main branch, and thus reverting the changes. You can then add and commit them to revert the files successfully.

Alternatively, you could use `git rebase` and remove the changed files in the earlier commit.

**Comment by calicokatsu** [comment]
Mind if I just reopen this?
